bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -48,30 +56,129 @@ def get_health(self, url: str):
             'url': url
         })
 
+    # TODO: this is not strictly following the 2.0 Core API Spec (or maybe the TIP isn't updated yet)
```

Comment:
We're doing this so users get the url which can be interesting if multiple node urls were provided to the client, makes no sense in the TIP (the value inside is following the TIP, so I don't think that's a problem)

Reply:
Ok, just reading the comment it sounds like the return type or TIP is wrong